### PR TITLE
Rename AIR DSL to AirScript throughout the project

### PR DESCRIPTION
**Pull request by tohrnii**
Addressing #59.

@bobbinth Should `Air` in `AirScript` be capitalized?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
-# AIR DSL
+# AIR SCRIPT
```
Comment (bobbinth) [inline README.md:1]: nit: I'd probably keep it as "AirScript" here.

File: docs/src/description/structure.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
 # File Structure
 
-An Air DSL file consists of a name definition and several source sections which contain declarations and constraints. The declarations describe the shape of the execution trace to which constraints are applied and the public inputs, and periodic columns that are used for computing those constraints. The constraints describe boundary and transition constraints which must hold for an execution trace and set of public inputs in order for them to be valid (i.e. in order for a valid proof to be generated or verification to pass).
+An AirScript file consists of a name definition and several source sections which contain declarations and constraints. The declarations describe the shape of the execution trace to which constraints are applied and the public inputs, and periodic columns that are used for computing those constraints. The constraints describe boundary and transition constraints which must hold for an execution trace and set of public inputs in order for them to be valid (i.e. in order for a valid proof to be generated or verification to pass).
 
 ## Air name definition
```
Comment (bobbinth) [inline docs/src/description/structure.md:5]: nit: I'd probably capitalize "AIR" here.

File: docs/src/introduction.md [hunk 3]
```diff
@@ -18,24 +18,24 @@ The language will be specified in detail in the rest of this book.
...
 ```
 cargo build --release
 ```
 
-Then, run the `air-dsl` target with the `transpile` option and specify your input file with `-i`. For example:
+Then, run the `air-script` target with the `transpile` option and specify your input file with `-i`. For example:
 
 ```
-./target/release/air-dsl transpile -i examples/system.air
+./target/release/air-script transpile -i examples/system.air
```
Comment (bobbinth) [inline docs/src/introduction.md:32]: Thought for the future: I wonder if we should rename the executable to just `air`.

**Review by bobbinth — APPROVED** [review]
Looks good to me! Thank you! I left a couple of small nits inline, but I'm approving the PR even without them.

And yes, let's stick with "AirScript".

File: docs/src/introduction.md (hunk 3 above)
Comment (tohrnii) [reply docs/src/introduction.md:32]: I've moved this comment to #1 for discussion.

**Pull request merged**